validation/analyze_qam.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+""" Range quality metric analysis to plot statistics and generate Latex tables
+
+    run example: run analyze_qam.py input_path result_path
+
+    output:
+            png figure with mean values, std and mean signed difference
+            txt file with LATEX table
+
+"""
+import argparse
+import glob
+import json
+import os
+
+import latextable
+import matplotlib
+matplotlib.use('agg')
+import matplotlib.pyplot as plt
+import numpy as np
+from tabulate import tabulate
+from texttable import Texttable
+import json
+
+def parse_run_experiment_args():
+    """ Parse input args for run_experiment_* scripts """
+    parser = argparse.ArgumentParser(description='Range quality assessment aggregated statistics from json files')
+    parser.add_argument('json_path')
+    parser.add_argument('out_dir')
+    parser.add_argument('-c', '--cores', default=2, type=int,
+                        help='Number of cores for parallel computation')
+    return parser.parse_args()
+
+def generate_latex_table(data_esa, data_nersc, out_path, data_name, print_RQM = True):
```

Comment:
It is better to split this function in two:
compute_mean_rqm
and
generate_latex_table
It will be much cleaner and more understandable.
Also, can you remove print_RQM? It controls not only printing but saving to file. And a lot of functionality is duplicated. Very unclear what is the role of that flag.

Reply:
That will also give you possibility to make one plot with 5 areas in Python (rather than in latex)